centos_ci.sh

```diff
@@ -292,14 +292,40 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  DOC_USER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Fetch remote repo and checkout to specified branch if not default minishift repo
+  if [[ "$DOC_USER" != "minishift" ]]; then
```

Comment:
This seems odd, should the ability to specify a different repo not be baked in at the top level, instead of just for building the docs?

Reply:
And how is this all used? There seem to be no docs either.